ms2pip/models/HCD/hcd_fast_B.c

```diff
@@ -1,3046 +0,0 @@
-static float score_HCD_B(unsigned int* v){
```

Comment:
`ms2pip/cython_modules/ms2pipfeatures_c_HCDch2.c` contains a reference (in comments) to these deleted files, that should be removed as well.